apps/dashboards/tests/test_api.py

```diff
@@ -92,3 +96,33 @@ def test_delete(self):
         url = reverse("dashboard-detail", kwargs={"pk": fake_data.id})
         response = self.client.delete(url)
         assert response.status_code == 204
+
+
+class Test_Dashboard_and_VisualizationViewSet_non_Staff_Request(
+    LoggedInMixin, APITestCase
+):
+    def setUp(self):
+        self.factory = APIRequestFactory()
+        super().setUp()
+
+    def test_dashboard_list_view_as_non_staff(self):
```

Comment:
This belongs to the `TestDashboardViewSet` test case.

Reply:
Fixed